tests/worker/test_workflow.py

```diff
@@ -5628,11 +5579,11 @@ async def test_unfinished_handler_on_workflow_termination(
         "-cancellation-", "-failure-", "-continue-as-new-"
     ],
 ):
-    skip_unfinished_handler_tests_in_older_python()
-    if handler_type == "-update-" and env.supports_time_skipping:
+    if env.supports_time_skipping:
         pytest.skip(
-            "Java test server: https://github.com/temporalio/sdk-java/issues/1903"
+            "Issues with update: https://github.com/temporalio/sdk-python/issues/826"
```

Comment:
FYI @dandavison - I opened https://github.com/temporalio/sdk-python/issues/826 to investigate why we can't unskip this yet